dkg/frost.go

```diff
@@ -219,7 +219,7 @@ func makeShares(
 	targetID uint32,
 ) ([]share, error) {
 	// Get set of public shares for each validator.
-	pubShares := make(map[uint32]map[uint32]*bls_sig.PublicKey) // map[ValIdx]map[SourceID]*bls_sig.PublicKey
+	pubShares := make(map[uint32]map[int]*bls_sig.PublicKey) // map[ValIdx]map[SourceID]*bls_sig.PublicKey
```

Comment:
nit: why `int` instead of `uint32`?

Reply:
while going through the code i didn't found any specific reason to use uint32 for dkg, that's why i went with fundamental types such as int

can change if there was any specific reason behind the usage of uint32 for dkg purposes